Parent/Parent/Dashboard/DashboardViewController.swift

```diff
@@ -315,17 +303,7 @@ class DashboardViewController: UIViewController {
     }
     
     @objc func showNotAParentView() {
-        let vc =  HelmViewController( moduleName: "/parent/notAParent", props: [:] )
-        showViewController(vc)
-    }
-    
-    //  MARK: - Helpers
-    @objc func showViewController(_ viewController: UIViewController) {
-        viewController.willMove(toParent: self)
-        addChild(viewController)
-        view.addSubview(viewController.view)
-        viewController.didMove(toParent: self)
-        viewController.view.pinToAllSidesOfSuperview()
+        AppEnvironment.shared.router.route(to: .wrongApp, from: self, options: [.modal, .embedInNav])
```

Comment:
We try to have the presenter do the routing since. It will be given the `AppEnvironment` and that way we can (one day) test that the route gets called.